lib/Bio/EnsEMBL/DataCheck/Checks/MySQLStorageEngine.pm

```diff
@@ -38,7 +38,7 @@ use constant {
 sub tests {
   my ($self) = @_;
   my $database_name = $self->dba->dbc->dbname;
-  my $engine = 'MyISAM';
+  my $engine = ($database_name =~ /_compara_/ and $database_name !~ /ensembl/) ? 'InnoDB' : 'MyISAM';
```

Comment:
Not sure about this one, the MyISAM storage engine is compulsory for subsequent processing, why should we accept `innoDB` for compara particularly?

Reply:
This is specific to the rapid release compara databases and not "main" release compara databases. When you say subsequent processing - what exactly do you mean, what subsequent processing is performed on the per species RR compara dbs? The decision for these dbs has been to be `InnoDB` for a while - and by default for RR they were on our side, but in the handover process, this seems to have been lost. In testing, there did not appear to be any issues with the use of `InnoDB` with web - but perhaps they did not notice?

Reply:
A further note on the use of `InnoDB` in the per species compara dbs instead of `MyISAM` is that the schema takes advantage of the `INDEX`, `FOREIGN KEYS` and `CASCADE`, in some ways that the "main" release compara db cannot - with `MyISAM`

Reply:
Sorry :sweat_smile: , that was not intended to teach you the benefits of `InnoDB` over `MyISAM` - I was trying to highlight that the specific features of `InnoDB` in the RR compara db depends on these in ways that the "main" release db at present does not. Here I filter on the name to match `compara` because this is a key word in the compara dbs, but exclude the use of `ensembl` - because this is in the nomenclature of the main release compara dbs - so "main" release dbs *should* not slip through.

Reply:
ok got it.